code/modules/mob/language/station_vr.dm

```diff
@@ -23,4 +23,18 @@
 	speech_verb = "snarls"
 	colour = "sergal"
 	key = "z"
-	syllables = list ("grr", "gah", "woof", "arf", "arra", "rah", "wor", "sarg")
\ No newline at end of file
+	syllables = list ("grr", "gah", "woof", "arf", "arra", "rah", "wor", "sarg")
+
+/datum/language/vulpkanin
+	name = "Canilunzt"
+	desc = "The guttural language spoken and utilized by the inhabitants of Vazzend system, composed of growls, barks, yaps, and heavy utilization of ears and tail movements.Vulpkanin speak this language with ease."
```

Comment:
Minor grammar issue here.

"... utilization of ears and tail movements.Vulpkanin speak this ..."

There needs to be a space there.